package.json

```diff
@@ -45,7 +45,7 @@
     "nock": "^8.0.0",
     "sinon": "^1.17.4",
     "tslint": "^3.15.1",
-    "typescript": "^2.0.3"
+    "typescript": "^2.4.1"
```

Comment:
This isn't strictly necessary (or at least it can be postponed until after this release). When NPM run build runs, it uses the maximum version that is available. If we raise this version then our TypeScript customers must upgrade too.